src/api/app/views/webui2/webui/projects/rebuild_times/show.html.haml

```diff
@@ -0,0 +1,33 @@
+:ruby
+  content_for(:content_for_head, javascript_include_tag('webui2/cm2/index-prjconf'))
+  @pagetitle = "Rebuild Time for Project #{@project}"
+  @metarobots = 'noindex'
+  p1 = @longestpaths[0].reverse
+  p2 = @longestpaths[1].reverse
+  p3 = @longestpaths[2].reverse
+
+.card
+  = render(partial: 'webui/project/tabs', locals: { project: @project })
+  .card-body
+    %h3
+      Rebuild Time: #{distance_of_time_in_words(Time.now, Time.now - @rebuildtime)} (#{format_seconds(@rebuildtime)})
+    %p
+      %img.rebuildgraph{ alt: 'Rebuild graph', src: project_rebuild_time_png_path(project: @project, key: @pngkey) }/
```

Comment:
The alt should be: Rebuild **G**raph

Reply:
> The alt should explain that it couldn't be loaded. Because right now, it doesn't say much.

`alt` changed to: `'The image could not be loaded. This should be a Rebuild Graph'`.

Reply:
Nice. I fixed the second sentence due to a missing word and unneeded capitalization (so it's now `This should be a rebuild time graph`).